specification/contosowidgetmanager/Contoso.WidgetManager/tspconfig.yaml

```diff
@@ -11,21 +11,24 @@ options:
   "@azure-tools/typespec-autorest":
     azure-resource-provider-folder: "data-plane"
     emitter-output-dir: "{project-root}/.."
-    examples-directory: "examples"
+    examples-directory: "{project-root}/examples"
     output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/widgets.json"
   "@azure-tools/typespec-python":
     package-dir: "azure-contoso-widgetmanager"
     package-mode: "dataplane"
     package-name: "{package-dir}"
   "@azure-tools/typespec-csharp":
-    package-dir: "Azure.Contoso.WidgetManager"
+    package-dir: "Azure.Storage.Contoso"
```

Comment:
Talked to @m-nash and agreed to use an approved namespace to succeed the build.

Reply:
Should we use Azure.Template here instead? https://github.com/Azure/azure-sdk-tools/blob/main/src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers/ClientAssemblyNamespaceAnalyzer.cs#L34C14-L34C28